storybook-docs/.storybook/main.ts

```diff
@@ -0,0 +1,55 @@
+/** @type { import('@storybook/react-vite').StorybookConfig } */
+import type { StorybookConfig } from '@storybook/react-vite';
+
+const config: StorybookConfig = {
+    stories: [
+        "../stories/**/*.mdx",
+        "../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)",
+    ],
+    addons: [
+        "@storybook/addon-onboarding",
+        "@storybook/addon-links",
+        "@storybook/addon-essentials",
+        "@chromatic-com/storybook",
+        "@storybook/addon-interactions",
+    ],
+    framework: {
+        name: "@storybook/react-vite",
+        options: {},
+    },
+    docs: {
+        autodocs: "tag",
+        defaultName: "Documentation",
```

Comment:
```suggestion
```

I think the default `Docs` is fine?

Reply:
Copywriters (Pei and Isabelle) are not a big fan of abbreviation so I would avoid it

Reply:
I agree with @SamuelAlev here, so i'd sugest we keep it

Reply:
makes sense, thanks for the context!